.github/workflows/ci.yml

```diff
@@ -21,54 +21,43 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'lts'
           - '1' # automatically expands to the latest stable 1.x release of Julia
           - 'nightly'
         os:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
         arch:
-          - x64
+          - default
```

Comment:
Can be dropped?

Reply:
It can't because we use arch for an included x86 so it needs to be populated with the default